cmd/promtool/testdata/rules.yml

```diff
@@ -25,4 +25,4 @@ groups:
 
       # Subquery with default resolution test.
       - record: suquery_interval_test
-        expr: count_over_time(up[5m:])
+        expr: count_over_time(up[5m1ms:])
```

Comment:
It would actually be more interesting to leave this at `5m` and make sure that the result is one lower than before.

Reply:
Seem not to change any behavior, so I undo this change.